bootstrap.env

```diff
@@ -0,0 +1,29 @@
+export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
+export CONJUR_MINOR_VERSION=5.0
+export CONJUR_APPLIANCE_IMAGE=registry.tld/conjur-appliance:$CONJUR_MINOR_VERSION-stable
+export CONJUR_FOLLOWER_COUNT=1
+export CONJUR_ACCOUNT=account-$UNIQUE_TEST_ID
+export AUTHENTICATOR_ID=conjur-$UNIQUE_TEST_ID-test
+export CONJUR_ADMIN_PASSWORD=ADmin123!!!!
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_NAMESPACE_NAME=conjur-$UNIQUE_TEST_ID
+export TEST_RUNNER_IMAGE=demo-$UNIQUE_TEST_ID
```

Comment:
why does this start with `demo`?

Reply:
I didn't add this but I see you named it `TEST_RUNNER_IMAGE`. Where did this naming come from and why is not `JENKINS_IMAGE` because we are only using this TEST RUNNER when we run a build in Jenkins.

Reply:
no idea, i probably took it as is from another repo. feel free to change it to whatever seems right

Reply:
but i wouldn't call it `JENKINS_IMAGE` because that seems to be an image of Jenkins. I think that `TEST_RUNNER_IMAGE` is not bad, becaese the image built with this tag will run the tests.